### ISSUE TITLE
2019-11-21 tc018 producing test code output and not producing partial AURC results

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#52]
Link to #34 

## Partial AURCs requested 
Here it may not be possible to compute these if there's no midpoint specified for these. This will be checked against the specification.

```
  AUC.1.T1 AUC.1.T2
1        0       12
```    

## run_computation is generating test outputs inherited from the master branch

```
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
specific_gravity_adjustment : units not recognized!i:  1  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  1  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  2  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  2  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  3  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  3  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  4  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  4  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  5  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  5  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  6  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  6  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  7  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  7  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  8  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  8  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  9  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  9  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  10  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  10  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  11  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  11  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  12  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  12  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  13  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  13  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  14  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  14  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  15  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  15  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  16  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  16  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  17  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  17  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  18  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  18  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  19  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  19  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  20  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  20  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  21  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  21  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  22  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  22  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  23  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  23  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  24  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  24  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  25  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  25  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  26  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  26  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  27  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  27  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  28  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  28  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  29  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  29  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  30  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  30  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  31  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  31  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  32  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  32  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  33  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  33  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  34  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  34  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  35  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  35  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  36  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  36  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  37  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  37  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  38  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  38  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  39  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  39  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  40  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  40  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  41  tmp_dose:  200  d:  1  dose[[d]]:  200 
i:  41  tmp_dose:  200  d:  1  dose[[d]]:  200 
specific_gravity_adjustment : units not recognized!i:  42  tmp_dose:  30  d:  1  dose[[d]]:  30 
i:  42  tmp_dose:  30  d:  1  dose[[d]]:  30 
specific_gravity_adjustment : units not recognized!i:  43  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  43  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  44  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  44  tmp_dose:  100  d:  1  dose[[d]]:  100 
specific_gravity_adjustment : units not recognized!i:  45  tmp_dose:  100  d:  1  dose[[d]]:  100 
i:  45  tmp_dose:  100  d:  1  dose[[d]]:  100 
unit_conversion : Unit Class 1 (Time) time_col:  TLAG KELTMLO KELTMHI THALF MIDPT1 MIDPT2 TAU1 TOLD1  parameters are scaled from  HR  to  HR  via scaling factor:  1 
unit_conversion : Unit Class 2 (Amount) amount_col:  AMT.12.00 AMT.24.00 AE.12.00 AE.24.00 AE  parameters are scaled from  NG  to  MG  via scaling factor:  1e-06 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE1  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 4 (Volume) volume_col:  VOLSUM  parameters are scaled from  ML  to  L  via scaling factor:  0.001 
unit_conversion : Unit Class 6 (1/Time) kel_col:  KEL  parameters are scaled from  1/HR  to  1/HR  via scaling factor:  1 
unit_conversion : Unit Class 11: ([Volume.Amount]/Volume) aurc_col:  AURCALL AURCLAST AURCINFO AURCINFP  parameters are scaled from  ML.NG/ML  to  ML.UG/ML  via scaling factor:  0.001 
unit_conversion : Unit Class 13: (Amount/Time) rate_col:  MAXRATE1 RATELAST1 RATE1 RATE2  parameters are scaled from  NG/HR  to  UG/HR  via scaling factor:  0.001 
There were 50 or more warnings (use warnings() to see the first 50)
```

## Missing Partial AURC for 0
```
       SDEID AMT.12.00 AMT.24.00 AE.12.00 AE.24.00 AEPCT.12.00 AEPCT.24.00       AE    AEPCT  MAXRATE1 TMAXRATE1 RATELAST1 MIDPTLAST1 TLAG KEL KELTMLO KELTMHI KELNOPT KELR KELRSQ KELRSQA FLGACCEPTKEL THALF THALFF AURCALL AURCLAST AURCINFO AURCINFP
1  102652097   79.3760   27.7380  79.3760 107.1140    39688000    53557000 107.1140 53557000 6614.6667         6 2311.5000         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
2  155315252   41.3270   22.2250  41.3270  63.5520    20663500    31776000  63.5520 31776000 3443.9167         6 1852.0833         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
3  212128320   66.4200   35.6890  66.4200 102.1090    33210000    51054500 102.1090 51054500 5535.0000         6 2974.0833         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
4  232007206   43.4750   34.0375  43.4750  77.5125    43475000    77512500  77.5125 77512500 3622.9167         6 3622.9167          6   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
5  303069272    4.3056    3.0774   4.3056   7.3830    14352000    24610000   7.3830 24610000  358.8000         6  256.4500         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
6  322184155   44.7300    4.6200  44.7300  49.3500    22365000    24675000  49.3500 24675000 3727.5000         6  385.0000         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
7  346821548    8.5995    1.8480   8.5995  10.4475    28665000    34825000  10.4475 34825000  716.6250         6  154.0000         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
8  359622627   43.5400   34.5466  43.5400  78.0866    21770000    39043300  78.0866 39043300 3628.3333         6 2878.8833         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
9  392430764   35.1200   28.1775  35.1200  63.2975    35120000    63297500  63.2975 63297500 2926.6667         6 2926.6667          6   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
10 395827329   62.4800   28.0416  62.4800  90.5216    31240000    45260800  90.5216 45260800 5206.6667         6 2336.8000         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
11  43456658    6.3280    3.7180   6.3280  10.0460    21093333    33486667  10.0460 33486667  527.3333         6  309.8333         18   NA  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>      NA       NA       NA       NA
   AURCXPCTO AURCXPCTP VOLSUM RATE1 RATE2 MIDPT1 MIDPT2 DI1 TAU1 TOLD1 DOSE1 TIMEU AMOUNTU DOSEU VOLUMEU CONCU KELU CLU AUCU AUMCU AUCNORMU    AURCU CONCNORMU RATEU VOLUMEWU CLWU    STUDY SITEID   SUBJID RAND                              TREATXT
1         NA        NA  3.760    NA    NA      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011198 1056     PF-05175157- 200mg - T2DM cohort
2         NA        NA  3.085    NA    NA      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011132 1053     PF-05175157- 200mg - T2DM cohort
3         NA        NA  2.940    NA    NA      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011219 1062     PF-05175157- 200mg - T2DM cohort
4         NA        NA  2.725    NA    NA      6     18  NA   12     0   100    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011130 1042 PF-05175157- 100 mg  BID- HOV Cohort
5         NA        NA  2.160    NA    NA      6     18  NA   24     0    30    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011007 1006      PF-05175157- 30mg -  HOV Cohort
6         NA        NA  2.083    NA    NA      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011048 1026     PF-05175157- 200mg -  HOV Cohort
7         NA        NA  1.190    NA    NA      6     18  NA   24     0    30    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011023 1011      PF-05175157- 30mg -  HOV Cohort
8         NA        NA  2.166    NA    NA      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011078 1033     PF-05175157- 200mg -  HOV Cohort
9         NA        NA  1.775    NA    NA      6     18  NA   12     0   100    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011128 1040 PF-05175157- 100 mg  BID- HOV Cohort
10        NA        NA  1.608    NA    NA      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011086 1036     PF-05175157- 200mg -  HOV Cohort
11        NA        NA  1.210    NA    NA      6     18  NA   24     0    30    HR      MG    MG       L NG/ML 1/HR  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011011 1008      PF-05175157- 30mg -  HOV Cohort
   TRTCD   PKCOLL PKBDFLD      PKTERM PERIODU PERIOD VISITU VISIT PHASE     HT WT AGEDERU AGEDER WTUNI WTRAW HTUNI HTRAW RACEOTH RACES SEX RACIALD ETHNIC                               ACTTRT ACTTRTS ACTTRTC PKPCOM
1      F INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 168.91 NA      YR     54    NA    NA    IN  66.5      NA     W   M      NA     NA     PF-05175157- 200mg - T2DM cohort       F      NA   <NA>
2      F INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 167.89 NA      YR     43    NA    NA    IN  66.1      NA     W   M      NA     NA     PF-05175157- 200mg - T2DM cohort       F      NA   <NA>
3      F INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 169.93 NA      YR     48    NA    NA    IN  66.9      NA     W   M      NA     NA     PF-05175157- 200mg - T2DM cohort       F      NA   <NA>
4      E INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 174.50 NA      YR     41    NA    NA    IN  68.7      NA     W   M      NA     NA PF-05175157- 100 mg  BID- HOV Cohort       E      NA   <NA>
5      B INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 175.26 NA      YR     22    NA    NA    IN  69.0      NA     W   M      NA     NA      PF-05175157- 30mg -  HOV Cohort       B      NA   <NA>
6      D INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 170.94 NA      YR     32    NA    NA    IN  67.3      NA     W   M      NA     NA     PF-05175157- 200mg -  HOV Cohort       D      NA   <NA>
7      B INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 175.77 NA      YR     40    NA    NA    IN  69.2      NA     W   M      NA     NA      PF-05175157- 30mg -  HOV Cohort       B      NA   <NA>
8      D INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 162.56 NA      YR     43    NA    NA    IN  64.0      NA     W   M      NA     NA     PF-05175157- 200mg -  HOV Cohort       D      NA   <NA>
9      E INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 175.77 NA      YR     31    NA    NA    IN  69.2      NA     W   M      NA     NA PF-05175157- 100 mg  BID- HOV Cohort       E      NA   <NA>
10     D INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 160.78 NA      YR     34    NA    NA    IN  63.3      NA     W   M      NA     NA     PF-05175157- 200mg -  HOV Cohort       D      NA   <NA>
11     B INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 170.94 NA      YR     33    NA    NA    IN  67.3      NA     W   M      NA     NA      PF-05175157- 30mg -  HOV Cohort       B      NA   <NA>
 [ reached 'max' / getOption("max.print") -- omitted 34 rows ]
```
## Estimated concentration dataset is blank. This is expected since there's no selected terminal elimination phase datapoints. 

```
[1] PKDATAROWID SDEID       TIME        CEST_KEL    CEST_INT    CEST_EXT    CEST_C0     CEST_TLAST 
<0 rows> (or 0-length row.names)

```

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
Latest commit, 96f6417, still isn't generating the AURCs specified. At a minimum, it is expected that the AURC6_18 should be generated but doesn't appear in the generated parameter results.

## tc018 execute model 

```
  AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2
1        0       12        6       18
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
unit_conversion : Unit Class 1 (Time) time_col:  TMAXRATE TLAG KELTMLO KELTMHI THALF MIDPT1 MIDPT2 TAU1 TOLD1  parameters are scaled from  HR  to  HR  via scaling factor:  1 
unit_conversion : Unit Class 2 (Amount) amount_col:  AMT.12.00 AMT.24.00 AE.12.00 AE.24.00 AE  parameters are scaled from  NG  to  MG  via scaling factor:  1e-06 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE1  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 4 (Volume) volume_col:  VOLSUM  parameters are scaled from  ML  to  L  via scaling factor:  0.001 
unit_conversion : Unit Class 6 (1/Time) kel_col:  KEL  parameters are scaled from  1/HR  to  1/HR  via scaling factor:  1 
unit_conversion : Unit Class 11: ([Volume.Amount]/Volume) aurc_col:  AURCALL AURCLAST AURCINFO AURCINFP  parameters are scaled from  ML.NG/ML  to  ML.UG/ML  via scaling factor:  0.001 
unit_conversion : Unit Class 13: (Amount/Time) rate_col:  MAXRATE MAXRATE1 RATELAST1 RATE1 RATE2  parameters are scaled from  NG/HR  to  UG/HR  via scaling factor:  0.001 
Warning messages:
1: In run_M4_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
2: In run_M4_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTPREDOSECRIT' is not present in the dataset
3: In run_M4_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' is not present in the dataset
4: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'PKPTMU', 'CONCU' and 'DOSEU' #8# are not present in the dataset provided via 'map'
5: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'CONCU' #24# is not present in the dataset provided via 'map'
6: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'CONCU' and 'NORMBSU' #14# are not present in the dataset provided via 'map'
7: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'CONCU', 'PKPTMU and 'NORMBSU' #19# are not present in the dataset provided via 'map'
> r <- results_list$data_out
> e <- results_list$est_data
> f <- results_list$flag_data

```

## Parameter results - missing AURCs requested through mct specification
```
> r
       SDEID AMT.12.00 AMT.24.00 AE.12.00 AE.24.00 AEPCT.12.00 AEPCT.24.00       AE    AEPCT   MAXRATE TMAXRATE  MAXRATE1 TMAXRATE1
1  102652097   79.3760   27.7380  79.3760 107.1140    39688000    53557000 107.1140 53557000 6614.6667        6 6614.6667         6
2  155315252   41.3270   22.2250  41.3270  63.5520    20663500    31776000  63.5520 31776000 3443.9167        6 3443.9167         6
3  212128320   66.4200   35.6890  66.4200 102.1090    33210000    51054500 102.1090 51054500 5535.0000        6 5535.0000         6
4  232007206   43.4750   34.0375  43.4750  77.5125    43475000    77512500  77.5125 77512500 3622.9167        6 3622.9167         6
5  303069272    4.3056    3.0774   4.3056   7.3830    14352000    24610000   7.3830 24610000  358.8000        6  358.8000         6
6  322184155   44.7300    4.6200  44.7300  49.3500    22365000    24675000  49.3500 24675000 3727.5000        6 3727.5000         6
7  346821548    8.5995    1.8480   8.5995  10.4475    28665000    34825000  10.4475 34825000  716.6250        6  716.6250         6
8  359622627   43.5400   34.5466  43.5400  78.0866    21770000    39043300  78.0866 39043300 3628.3333        6 3628.3333         6
9  392430764   35.1200   28.1775  35.1200  63.2975    35120000    63297500  63.2975 63297500 2926.6667        6 2926.6667         6
10 395827329   62.4800   28.0416  62.4800  90.5216    31240000    45260800  90.5216 45260800 5206.6667        6 5206.6667         6
11  43456658    6.3280    3.7180   6.3280  10.0460    21093333    33486667  10.0460 33486667  527.3333        6  527.3333         6
   RATELAST1 MIDPTLAST1 TLAG KEL KELTMLO KELTMHI KELNOPT KELR KELRSQ KELRSQA FLGACCEPTKEL THALF THALFF   AURCALL  AURCLAST AURCINFO
1  2311.5000         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA> 49113.902 49113.902       NA
2  1852.0833         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA> 30794.863 30794.863       NA
3  2974.0833         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA> 49473.908 49473.908       NA
4  3622.9167          6    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA> 38563.977 38563.977       NA
5   256.4500         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>  3657.192  3657.192       NA
6   385.0000         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA> 17667.660 17667.660       NA
7   154.0000         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>  4390.934  4390.934       NA
8  2878.8833         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA> 38870.054 38870.054       NA
9  2926.6667          6    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA> 31521.431 31521.431       NA
10 2336.8000         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA> 42985.811 42985.811       NA
11  309.8333         18    6  NA      NA      NA       0   NA     NA      NA            0    NA   <NA>  4907.875  4907.875       NA
   AURCINFP AURCXPCTO AURCXPCTP VOLSUM     RATE1     RATE2 MIDPT1 MIDPT2 DI1 TAU1 TOLD1 DOSE1 TIMEU AMOUNTU DOSEU VOLUMEU CONCU KELU
1        NA        NA        NA  3.760 6614.6667 2311.5000      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR
2        NA        NA        NA  3.085 3443.9167 1852.0833      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR
3        NA        NA        NA  2.940 5535.0000 2974.0833      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR
4        NA        NA        NA  2.725 3622.9167 2836.4583      6     18  NA   12     0   100    HR      MG    MG       L NG/ML 1/HR
5        NA        NA        NA  2.160  358.8000  256.4500      6     18  NA   24     0    30    HR      MG    MG       L NG/ML 1/HR
6        NA        NA        NA  2.083 3727.5000  385.0000      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR
7        NA        NA        NA  1.190  716.6250  154.0000      6     18  NA   24     0    30    HR      MG    MG       L NG/ML 1/HR
8        NA        NA        NA  2.166 3628.3333 2878.8833      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR
9        NA        NA        NA  1.775 2926.6667 2348.1250      6     18  NA   12     0   100    HR      MG    MG       L NG/ML 1/HR
10       NA        NA        NA  1.608 5206.6667 2336.8000      6     18  NA   24     0   200    HR      MG    MG       L NG/ML 1/HR
11       NA        NA        NA  1.210  527.3333  309.8333      6     18  NA   24     0    30    HR      MG    MG       L NG/ML 1/HR
   CLU AUCU AUMCU AUCNORMU    AURCU CONCNORMU RATEU VOLUMEWU CLWU    STUDY SITEID   SUBJID RAND                              TREATXT
1   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011198 1056     PF-05175157- 200mg - T2DM cohort
2   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011132 1053     PF-05175157- 200mg - T2DM cohort
3   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011219 1062     PF-05175157- 200mg - T2DM cohort
4   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011130 1042 PF-05175157- 100 mg  BID- HOV Cohort
5   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011007 1006      PF-05175157- 30mg -  HOV Cohort
6   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011048 1026     PF-05175157- 200mg -  HOV Cohort
7   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011023 1011      PF-05175157- 30mg -  HOV Cohort
8   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011078 1033     PF-05175157- 200mg -  HOV Cohort
9   NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011128 1040 PF-05175157- 100 mg  BID- HOV Cohort
10  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011086 1036     PF-05175157- 200mg -  HOV Cohort
11  NA   NA    NA       NA ML.UG/ML        NA UG/HR       NA   NA B1731007   1001 10011011 1008      PF-05175157- 30mg -  HOV Cohort
   TRTCD   PKCOLL PKBDFLD      PKTERM PERIODU PERIOD VISITU VISIT PHASE     HT WT AGEDERU AGEDER WTUNI WTRAW HTUNI HTRAW RACEOTH RACES
1      F INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 168.91 NA      YR     54    NA    NA    IN  66.5      NA     W
2      F INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 167.89 NA      YR     43    NA    NA    IN  66.1      NA     W
3      F INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 169.93 NA      YR     48    NA    NA    IN  66.9      NA     W
4      E INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 174.50 NA      YR     41    NA    NA    IN  68.7      NA     W
5      B INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 175.26 NA      YR     22    NA    NA    IN  69.0      NA     W
6      D INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 170.94 NA      YR     32    NA    NA    IN  67.3      NA     W
7      B INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 175.77 NA      YR     40    NA    NA    IN  69.2      NA     W
8      D INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 162.56 NA      YR     43    NA    NA    IN  64.0      NA     W
9      E INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 175.77 NA      YR     31    NA    NA    IN  69.2      NA     W
10     D INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 160.78 NA      YR     34    NA    NA    IN  63.3      NA     W
11     B INTERVAL   URINE PF-05175157      NA     NA    DAY    14    NA 170.94 NA      YR     33    NA    NA    IN  67.3      NA     W
   SEX RACIALD ETHNIC                               ACTTRT ACTTRTS ACTTRTC PKPCOM
1    M      NA     NA     PF-05175157- 200mg - T2DM cohort       F      NA   <NA>
2    M      NA     NA     PF-05175157- 200mg - T2DM cohort       F      NA   <NA>
3    M      NA     NA     PF-05175157- 200mg - T2DM cohort       F      NA   <NA>
4    M      NA     NA PF-05175157- 100 mg  BID- HOV Cohort       E      NA   <NA>
5    M      NA     NA      PF-05175157- 30mg -  HOV Cohort       B      NA   <NA>
6    M      NA     NA     PF-05175157- 200mg -  HOV Cohort       D      NA   <NA>
7    M      NA     NA      PF-05175157- 30mg -  HOV Cohort       B      NA   <NA>
8    M      NA     NA     PF-05175157- 200mg -  HOV Cohort       D      NA   <NA>
9    M      NA     NA PF-05175157- 100 mg  BID- HOV Cohort       E      NA   <NA>
10   M      NA     NA     PF-05175157- 200mg -  HOV Cohort       D      NA   <NA>
11   M      NA     NA      PF-05175157- 30mg -  HOV Cohort       B      NA   <NA>
 [ reached 'max' / getOption("max.print") -- omitted 34 rows ]
```

## estimated concentration dataset is empty

```
> e
[1] PKDATAROWID SDEID       TIME        CEST_KEL    CEST_INT    CEST_EXT    CEST_C0     CEST_TLAST 
<0 rows> (or 0-length row.names)
```

**Comment by jhhughes256** [comment]
### Update

I have noticed when running tc018 in the latest commit 18821dca00 that the following warning is given. I have included it below in case it helps.

```
In run_computation(data = d, map = mct, flag = flags,  ... :
  Unable to generate dosing interval for Steady State data! TAU value 'TAU1' or TOLD value 'TOLD1' not found in the provided TIME data
```

**Comment by tensfeldt** [comment]
Closed with commit 22d43ea
